src/Exception/AwsException.php

```diff
@@ -81,6 +85,16 @@ public function getCommand()
     }
 
     /**
+     * Get the concise error message if any.
+     *
+     * @return string|null
+     */
+    public function getErrorMessage()
```

Comment:
Thanks for the suggestions. Have changed the method name to `getAwsErrorMessage`